aexpect/remote.py

```diff
@@ -19,6 +19,27 @@
 
 """
 Functions and classes used for logging into guests and transferring files.
+
+Example usage:
+
+::
+
+    import aexpect
+    import aexpect.remote
+
+    b = aexpect.ShellSession("sh")
+    b.cmd("echo asdf > /tmp/bbb")
+
+    a = aexpect.remote.remote_login("ssh", "127.0.0.1", 5000, "root", "123456", "[\\#\\$]")
```

Comment:
It'd be nice to use the built-in `aexpect.remote.PROMPT_LINUX`

Reply:
Done